metric_metadata/errors.go

```diff
@@ -11,12 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+package metadata
```

Comment:
nit: newline before this line otherwise godoc thinks the license is the package comment.